s1/Program copy.cs

```diff
@@ -0,0 +1,117 @@
+﻿int n_rows = 10;
+int n_cols = 30;
+int score = 0;
+string statusMessage = "";
+char[,] map = new char[n_rows, n_cols];
+var random = new Random();
```

Comment:
You try to be consistent between `var` and specific types. We prefer to use `var` at Instech. Of course, sometimes you still need to use explicit types if the variable is not initialized immediately.